PyInstaller/compat.py

```diff
@@ -215,7 +216,7 @@
 
 
 # Wine detection and support
-def is_wine_dll(filename):
+def is_wine_dll(filename: str | bytes | os.PathLike | int):
```

Comment:
Technically, those are all valid options to pass to the underlying `open()`, but in practice, it should be used only with `str` (and perhaps an `os.PathLike`).